system/jlib/jsocket.cpp

```diff
@@ -79,6 +79,7 @@
 #   define EPOLLRDHUP 0x2000
 #  endif
 #  define MAX_RET_EVENTS  5000 // max events returned from epoll_wait() call
+   static unsigned epoll_hdlPerThrd = (unsigned)-1;
```

Comment:
This means default # of handles is UINT_MAX right? If so, I think would be better to use UINT_MAX for clarity.